arborium-perl: Linking error with MSVC

`cargo build` fails on Windows with:

```
ucrt.lib(api-ms-win-crt-utility-l1-1-0.dll) : error LNK2005: bsearch already defined in libarborium_perl-....rlib(scanner.o)
fatal error LNK1169: one or more multiply defined symbols found
```

After a bit a research it looks like changing `void *bsearch(...)` to `static inline void *bsearch(...)`could solve this linker issue.

This does not occur on macOS or Linux.

--- a/langs/group-hazel/perl/def/grammar/bsearch.h
+++ b/langs/group-hazel/perl/def/grammar/bsearch.h
@@ -53,6 +53,12 @@ is respectively less than, matching, or greater than the array member.
 #include <stdio.h>
 #include <sys/types.h> /* size_t */
 
+#ifdef bsearch
+#undef bsearch
+#endif
+
+#define bsearch tsp_bsearch
+
 /*
  * Perform a binary search.
  *
@@ -69,8 +75,9 @@ is respectively less than, matching, or greater than the array member.
  * have to make lim 3, then halve, obtaining 1, so that we will only
  * look at item 3.
  */
-void *bsearch(const void *key, const void *base0, size_t nmemb, size_t size,
-              int (*compar)(const void *, const void *)) {
+static inline void *tsp_bsearch(
+    const void *key, const void *base0, size_t nmemb, size_t size,
+    int (*compar)(const void *, const void *)) {
   const char *base = (const char *)base0;
   int lim, cmp;
   const void *p;

